master/pkg/model/test_utils.go

```diff
@@ -37,7 +37,7 @@ func ExperimentModel(opts ...ExperimentModelOption) *Experiment {
 				RawMaxLength: &maxLength,
 			},
 		},
-		RawEntrypoint: ptrs.StringPtr("model_def:SomeTrialClass"),
+		RawEntrypoint: &expconf.EntrypointV0{RawEntrypoint: "model_def:SomeTrialClass"},
```

Comment:
Don't use versioned symbols (`EntrypointV0`) outside of the expconf package.  There's a `latest.go` where you should create an alias.